category_theory/category.lean

```diff
@@ -73,11 +73,11 @@ abbreviation small_category (C : Type u)     : Type (u+1) := category.{u u} C
 
 structure bundled (c : Type u → Type v) :=
 (α : Type u)
-[str : c α]
+(str : c α)
```

Comment:
Compare https://leanprover.zulipchat.com/#narrow/stream/116395-maths/subject/limits.202.2E0/near/148053105. The current `[]`s are redundant with `mk_ob`, and make it hard to construct objects when you want to provide `str` explicitly. I'm also open to other arrangements as long as both explicit and implicit (from class inference) specification of `str` is convenient.